src/components/atoms/CircularProfileImage/index.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+
+interface Props {
+  src: string | null;
+  /**
+    lg: "w-10 h-10",
+    md: "w-8 h-8",
+    sm: "w-6 h-6",
+   */
+  styleType?: "huge" | "lg" | "md" | "sm";
+}
+
+/**
+ * 원형 유저 프로필 이미지 컴포넌트.
+ * styleType으로 크기를 전달받는다
+ * huge: "w-20 h-20",
+ * lg: "w-10 h-10",
+ * md: "w-8 h-8",
+ * sm: "w-6 h-6",
+ * 기본 사이즈는 'md': 32*32px이다.
+ */
+function CircularProfileImage({ src, styleType = "md" }: Props) {
+  const styles = {
+    huge: "w-20 h-20",
+    lg: "w-10 h-10",
+    md: "w-8 h-8",
+    sm: "w-6 h-6",
+  };
+  return (
+    <div className={`profile-image rounded-full overflow-hidden ${styles[styleType]}`}>
+      <Image
+        alt="유저 프로필 이미지"
+        src={src || "/images/default_profile_image.png"}
+        width={100}
+        height={100}
+        style={{ objectFit: "cover" }}
```

Comment:
next/Image에 objectFit cover 속성이 있을거예요. 
여러분이 쓰시는 next 버전에도 유효한지는 한번 적용해보시면 될 것 같습니다!
만약 objectFit props를 줘도 타입 에러가 안 나면 사용해도 될 겁니다
```suggestion
        objectFit="cover"
```

Reply:
> next/Image에 objectFit cover 속성이 있을거예요. 여러분이 쓰시는 next 버전에도 유효한지는 한번 적용해보시면 될 것 같습니다! 만약 objectFit props를 줘도 타입 에러가 안 나면 사용해도 될 겁니다
objectFit 속성은 next 13 버전에서 legacy prop이라는 warning을 띄워서 style prop으로 지정해 주었습니다. 참고 링크: https://nextjs.org/docs/messages/next-image-upgrade-to-13

Reply:
아하 13버전에서 레거시가 되었군요! 알겠습니다!!